### ISSUE TITLE
Win32Reactor.spawnProcess is redundant

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5304]
|[<img alt="zseil's avatar" src="https://avatars.githubusercontent.com/u/421894?s=50" width="50" height="50">](https://github.com/zseil)| @zseil reported|
|-|-|
|Trac ID|trac#5304|
|Type|defect|
|Created|2011-10-10 08:28:54Z|
|Branch|https://github.com/twisted/twisted/tree/duplicate-spawnProcess-5304|

`Win32Reactor` inherits from `PosixReactorBase`, which already provides a usable `spawnProcess` implementation (in fact, the implementations are the same apart from cross platform support and conditional imports in `PosixReactorBase`). `Win32Reactor`'s `spawnProcess` implementation should be deleted.

<details><summary>Searchable metadata</summary>

```
trac-id__5304 5304
type__defect defect
reporter__zseil zseil
priority__normal normal
milestone__ 
branch__branches_duplicate_spawnProcess_5304 branches/duplicate-spawnProcess-5304
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1318235334000000 1318235334000000
changetime__1318796012000000 1318796012000000
version__None None
owner__exarkun exarkun
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [32874]) Branching to 'duplicate-spawnProcess-5304'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [32875]) Remove the `IReactorProcess` implementation from win32eventreactor; it already inherits one from posixbase.

refs [#5304](https://github.com/twisted/twisted/issues/5304)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/duplicate-spawnProcess-5304)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @exarkun|
|-|-|

Yay deleting code. Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [32884]) Merge duplicate-spawnProcess-5304

Author: exarkun
Reviewer: jesstess
Fixes: [#5304](https://github.com/twisted/twisted/issues/5304)

Remove the redundant implementation of `IReactorProcess.spawnProcess` from
`twisted/internet/win32eventreactor.py`.
